code/modules/mob/living/silicon/mommi/mommi_modules.dm

```diff
@@ -36,6 +36,9 @@
 	sensor_augs = list("Mesons", "Disable")
 
 	fix_modules()
+	
+	if(Holiday == HALLOWEEN)
```

Comment:
Not sure if this is the best place for this

Reply:
where do you feel it is best?